docs/reference/cli-reference/cli-ref-sources.md

```diff
@@ -87,6 +87,9 @@ where `<operation>` is one of *List, Add, Remove, Enable, Disable,* or *Update*,
 - **`-Verbosity [normal|quiet|detailed]`**
 
   Specifies the amount of detail displayed in the output: `normal` (the default), `quiet`, or `detailed`.
+  
+- **`-AllowInsecureConnections`**
+  Allows HTTP connections for adding or updating packages. Note: This method is not secure.
```

Comment:
Should we document which version of NuGet this is added? A lot of people are running old versions od nuget.exe